AcadeMeData/tests.py

```diff
@@ -1,3 +1,34 @@
-# from django.test import TestCase
-
-# Create your tests here.
+from django.test import TestCase
+from AcadeMeData.models import Users
+
+
+class TestUserModel(TestCase):
+
+    def test_create_user(self):
+        username = "username"
+        password = "password"
+        email = "user@example.com"
+        type = "S"
+        university = "RU"
+        degree = "CS"

```

Comment:
Consider using dictionary
user_data = {'username': "username", 'password': "password", 'email':"user@example.com", 'type':"S", 'university': "RU",
'degree':"CS"}

and then you can call your method
Users.create_user(*user_data)